create_pdfs.py

```diff
@@ -119,8 +104,8 @@ def main():
             )
             title = None
             if args.assign_title:
-                title = random.choice(stanford_cme_titles)[0]
-                print "assigning random title", name, title
+                title = args.assign_title
+                print "assigning specific title", name, title
```

Comment:
I was going to say "let's use the print function instead of the print statement" [1], but instead, we should drop this line entirely.

It was originally added because the title was being randomly assigned.
Now that it's explicitly what we pass in, we don't gain anything by printing it out again.

[1] http://stackoverflow.com/a/6239909